packages/block-editor/src/components/url-input/index.js

```diff
@@ -22,229 +22,211 @@ import { withInstanceId, withSafeTimeout, compose } from '@wordpress/compose';
 import { withSelect } from '@wordpress/data';
 import { isURL } from '@wordpress/url';
 
-class URLInput extends Component {
-	constructor( props ) {
-		super( props );
-
-		this.onChange = this.onChange.bind( this );
-		this.onFocus = this.onFocus.bind( this );
-		this.onKeyDown = this.onKeyDown.bind( this );
-		this.selectLink = this.selectLink.bind( this );
-		this.handleOnClick = this.handleOnClick.bind( this );
-		this.bindSuggestionNode = this.bindSuggestionNode.bind( this );
-		this.autocompleteRef = props.autocompleteRef || createRef();
-		this.inputRef = createRef();
-		this.updateSuggestions = debounce(
-			this.updateSuggestions.bind( this ),
-			200
-		);
-
-		this.suggestionNodes = [];
-
-		this.isUpdatingSuggestions = false;
-
-		this.state = {
-			suggestions: [],
-			showSuggestions: false,
-			selectedSuggestion: null,
-
-			suggestionsListboxId: '',
-			suggestionOptionIdPrefix: '',
-		};
-	}
+const REQUEST_DEBOUNCE_DELAY = 200;
+
+function URLInput( {
+	value,
+	onChange: parentOnChange,
+	label = '',
+	className = '',
+	isFullWidth,
+	instanceId,
+	placeholder = __( 'Paste URL or type to search' ),
+	disableSuggestions,
+	speak,
+	debouncedSpeak,
+	__experimentalRenderControl: experimentalRenderControl,
```

Comment:
We usually rename experimental 'things' without the prefix, to make less changes when they become stable. See `__experimentalFetchLinkSuggestions: fetchLinkSuggestions ` below.

Reply:
Updated in e8e07b6